core/rfb.js

```diff
@@ -2863,10 +2879,36 @@ export default class RFB extends EventTargetMixin {
                                              this._fbWidth, this._fbHeight);
     }
 
+    _updateBrowserWindows(width, height) {
```

Comment:
The RFB object isn't really aware of things outside of its container. So I think these things are better placed in the UI object.

That means it needs some more information, though. We require `width` and `height` read-only properties on the RFB object. And we also need a `resize` event to be fired.